test/langtools/tools/javac/diags/examples/StringTemplateNotProcessor.java

```diff
@@ -22,8 +22,6 @@
  */
 
  // key: compiler.note.preview.filename
- // key: compiler.err.cant.resolve.location.args
- // key: compiler.misc.location
```

Comment:
I think this error was rather confusing, e.g.
```
Simple2.java:3: error: cannot find symbol
        System.out.println(e."\{e}");
                             ^
  symbol:   method process(StringTemplate)
  location: class String
```
It mentions the `process` method while it's not visible in the code. It also always came directly after e.g.
```
Simple2.java:3: error: not a processor type: String
        System.out.println(e."\{e}");
```
so it basically just caused 2 errors when there actually only is 1.